config/web/field.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+use Composer\InstalledVersions;
+use Yiisoft\Form\Widget\Field;
+
+if (InstalledVersions::isInstalled('yiisoft/form') && $params['yiisoft/form']['bulma']['enabled'] === true) {
```

Comment:
The condition looks odd. I don't think there's a need for it. Even if form package isn't installed, it should work alright.

Reply:
I was afraid of error if Form is not installed. So, not required?